doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -127,17 +127,34 @@ This section provides detailed lists of changes by :ref:`application <applicatio
 nRF9160: Asset Tracker v2
 -------------------------
 
+  * Added:
+
+    * Added the following Kconfig options to set the threshold and timeout values:
+
+      * :ref:`CONFIG_DATA_ACCELEROMETER_ACT_THRESHOLD <CONFIG_DATA_ACCELEROMETER_ACT_THRESHOLD>`
+      * :ref:`CONFIG_DATA_ACCELEROMETER_INACT_THRESHOLD <CONFIG_DATA_ACCELEROMETER_INACT_THRESHOLD>`
+      * :ref:`CONFIG_DATA_ACCELEROMETER_INACT_TIMEOUT_SECONDS <CONFIG_DATA_ACCELEROMETER_INACT_TIMEOUT_SECONDS>`
+    * Data sampling is now performed when the device detects both activity and inactivity in passive mode, notified by the :c:enum:`SENSOR_EVT_MOVEMENT_INACTIVITY_DETECTED` event of the :ref:`sensor module <asset_tracker_v2_sensor_module>`.
+
   * Removed:
 
     * ``CONFIG_APP_REQUEST_GNSS_ON_INITIAL_SAMPLING`` option.
     * ``CONFIG_APP_REQUEST_NEIGHBOR_CELLS_DATA`` option.
     * ``CONFIG_EXTERNAL_SENSORS_ACTIVITY_DETECTION_AUTO`` option.
+    * ``CONFIG_DATA_ACCELEROMETER_THRESHOLD`` option.
+    * ``CONFIG_DATA_ACCELEROMETER_BUFFER_STORE`` option.
+    * ``CONFIG_DATA_ACCELEROMETER_BUFFER_COUNT`` option.
+    * ``SENSOR_EVT_MOVEMENT_DATA_READY`` event.
 
   * Updated:
 
     * The default value of the GNSS timeout in the application's :ref:`Real-time configurations <real_time_configs>` is now 30 seconds.
     * GNSS fixes are now published in PVT format instead of NMEA for nRF Cloud builds. To revert to NMEA, set the :ref:`CONFIG_GNSS_MODULE_NMEA <CONFIG_GNSS_MODULE_NMEA>` option.
     * The sensor module now forwards :c:enum:`SENSOR_EVT_MOVEMENT_ACTIVITY_DETECTED` and :c:enum:`SENSOR_EVT_MOVEMENT_INACTIVITY_DETECTED` events.
+    * The :ref:`Real-time configurations <real_time_configs>` can now configure the upper and lower thresholds for motion detection.
+      You can also configure the timeout after which the sensor reports inactivity.
```

Comment:
@maxd-nordic : If we are referring to the same Kconfig mentioned in line no:134, 135 and 136 here. We can rephrase the sentence as below and delete line no: 132-136. 
```suggestion
    * The :ref:`Real-time configurations <real_time_configs>` can now configure the upper and lower thresholds for motion detection.
      You can also configure the timeout after which the sensor reports inactivity. 
      The following Kconfig options set the threshold and timeout values:

      * :ref:`CONFIG_DATA_ACCELEROMETER_ACT_THRESHOLD <CONFIG_DATA_ACCELEROMETER_ACT_THRESHOLD>`
      * :ref:`CONFIG_DATA_ACCELEROMETER_INACT_THRESHOLD <CONFIG_DATA_ACCELEROMETER_INACT_THRESHOLD>`
      * :ref:`CONFIG_DATA_ACCELEROMETER_INACT_TIMEOUT_SECONDS <CONFIG_DATA_ACCELEROMETER_INACT_TIMEOUT_SECONDS>`
```